docs/compass/03-01-security.md

```diff
@@ -16,46 +16,47 @@ The following diagram represents the architecture of the security in Compass:
 
 ### Tenant Mapping Handler
 
-It is an OathKeeper [hydrator](https://github.com/ory/docs/blob/525608c65694539384b785355d293bc0ad00da27/docs/oathkeeper/pipeline/mutator.md#hydrator) handler responsible for mapping authentication session data to tenant. It is built into Director itself, as it uses the same database. It is implemented as a separate endpoint, such as `/tenant-mapping`.
+It is an OathKeeper [hydrator](https://github.com/ory/docs/blob/525608c65694539384b785355d293bc0ad00da27/docs/oathkeeper/pipeline/mutator.md#hydrator) handler responsible for mapping authentication session data to tenant. It is built into the Director itself, as it uses the same database. It is implemented as a separate endpoint, such as `/tenant-mapping`.
 
 To unify the approach for mapping, we introduce `authorization_id`, widely used by multiple authentication flows.
 The `authorization_id` is equal to:
-- `client_id` in OAuth 2.0 authentication flow
-- `username` in Basic authentication flow
-- Common Name (CN) in Certificates authentication flow
+- `client_id` in the OAuth 2.0 authentication flow
+- `username` in the Basic authentication flow
```

Comment:
```suggestion
- `username` in the basic authentication flow
```

Reply:
Basic is the proper name here and should be capitalized.